### PR TITLE
Add rows for district heating and cooling virtual rate in LEED EAp2-3

### PR DESCRIPTION
**Pull request by JasonGlazer**
Add virtual rate calculations for district heating and district cooling in EAp2-3 Energy Type Summary table to solve issue #5614 
- [x] Code style (parentheses padding, variable names)
- [x] Functional code review (it has to work!)
- [x] If defect, results of running current develop vs this branch should exhibit the fix
- [x] CI status: all green or justified
- [x] Performance: CI Linux results include performance check -- verify this
- [x] Unit Test(s)
- [x] Documentation changes in place
- [x] If output changes, including tabular output structure, add to output rules file for interfaces

….

### CODE REVIEW

**Comment by JasonGlazer** [comment]
No other documentation changes are needed beside the OutputChanges8-5-0-to8-6-0.md file.

File: src/EnergyPlus/EconomicTariff.cc [hunk 1]
```diff
@@ -3868,11 +3904,24 @@ namespace EconomicTariff {
...
+				PreDefTableEntry( pdchLeedEtsDemUnt, "District Heating", convDemStrings( distHeatUnits ) + demWindowStrings( distHeatDemWindowUnits ) );
+				PreDefTableEntry( pdchLeedEcsTotal, "District Heating", distHeatTotalCost, 2 );
+			}
 			// save the total costs for later to compute process fraction
 			LEEDelecCostTotal = elecTotalCost;
 			LEEDgasCostTotal = gasTotalCost;
-			LEEDothrCostTotal = otherTotalCost;
-			PreDefTableEntry( pdchLeedEcsTotal, "Total", elecTotalCost + gasTotalCost + otherTotalCost, 2 );
+			LEEDothrCostTotal = distCoolTotalCost + distHeatTotalCost + otherTotalCost;
+			PreDefTableEntry( pdchLeedEcsTotal, "Total", elecTotalCost + gasTotalCost + distCoolTotalCost + distHeatTotalCost + otherTotalCost, 2 );
```
Comment (Myoldmopar) [inline src/EnergyPlus/EconomicTariff.cc:3924]: So with this change, does it correct previous outputs?  It kinda looks like for district heating/cooling cases, previously the "Total" would have been incorrectly not including those rates, and now they will be correct?

Comment (JasonGlazer) [inline src/EnergyPlus/EconomicTariff.cc:3924]: That was the intention

**Comment by Myoldmopar** [comment]
OK, thanks @JasonGlazer.  I think I'll give @davidgoldwasser and @macumber a heads-up about this change, but it looks good to me.  Thanks.

**Pull request merged**